src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -84,9 +85,8 @@ export default class CancerStudyTreeData
 				cancerTypeId: name
 			});
 		}
-		//cancerTypes = [virtualCohortsCategory].concat(this.priorityCategories).concat(this.rootCancerType, cancerTypes);
-		//studies = virtualCohortStudies.concat(studies);
-		cancerTypes = this.priorityCategories.concat(this.rootCancerType, cancerTypes);
+		cancerTypes = [virtualCohortsCategory].concat(this.priorityCategories).concat(this.rootCancerType, cancerTypes);

```

Comment:
not too important but look at new es6 spread operator to do this kind of concatenation